jax/_src/cudnn/fused_attention_stablehlo.py

```diff
@@ -963,7 +968,6 @@ def dot_product_attention(query: Array,
     dropout_rate: Dropout rate.
     qkv_layout: Layout string, with supported formats being BTNH, BNTH, BSNH,
                 BNSH.
-    is_training: choose to save activation or not.
```

Comment:
AFAICT this API is not public, so removing a parameter is fine. Once it becomes public, we might need to decide on the API compatibility guarantees. See https://jax.readthedocs.io/en/latest/api_compatibility.html.